setup.py

```diff
@@ -44,7 +44,7 @@
     ],
     python_requires=">=3.7",
     install_requires=[
-        "filetype>=1.0.10,!=1.1.0",
+        "filetype>=1.2.0,!=1.1.0",
```

Comment:
```suggestion
        "filetype>=1.2.0",
```

`!=1.1.0` means not 1.1.0, which is obsolete when we ask >=1.2.0